tests/data_context/store/test_ge_cloud_store_backend.py

```diff
@@ -22,393 +6,25 @@
-
-    my_simple_checkpoint_config: CheckpointConfig = CheckpointConfig(
-        name="my_minimal_simple_checkpoint",
-        class_name="SimpleCheckpoint",
-        config_version=1,
-    )
-    my_simple_checkpoint_config_serialized = (
-        my_simple_checkpoint_config.get_schema_class()().dump(
-            my_simple_checkpoint_config
-        )
-    )
-
-    with mock.patch("requests.Session.post", autospec=True) as mock_post:
-        store_backend.set(("checkpoint", ""), my_simple_checkpoint_config_serialized)
-        mock_post.assert_called_with(
-            mock.ANY,  # requests.Session object
-            f"{CLOUD_DEFAULT_BASE_URL}organizations/51379b8b-86d3-4fe7-84e9-e1a52f4a414c/checkpoints",
-            json={
-                "data": {
-                    "type": "checkpoint",
-                    "attributes": {
-                        "organization_id": "51379b8b-86d3-4fe7-84e9-e1a52f4a414c",
-                        "checkpoint_config": OrderedDict(
-                            [
-                                ("name", "my_minimal_simple_checkpoint"),
-                                ("config_version", 1.0),
-                                ("template_name", None),
-                                ("module_name", "great_expectations.checkpoint"),
-                                ("class_name", "SimpleCheckpoint"),
-                                ("run_name_template", None),
-                                ("expectation_suite_name", None),
-                                ("batch_request", {}),
-                                ("action_list", []),
-                                ("evaluation_parameters", {}),
-                                ("runtime_configuration", {}),
-                                ("validations", []),
-                                ("profilers", []),
-                                ("ge_cloud_id", None),
-                                ("expectation_suite_ge_cloud_id", None),
-                            ]
-                        ),
-                    },
-                }
-            },
-        )
-
-
-@pytest.mark.cloud
-@pytest.mark.unit
-def test_list_keys(
-    construct_ge_cloud_store_backend: Callable[
-        [GXCloudRESTResource], GeCloudStoreBackend
-    ],
-) -> None:
-    store_backend = construct_ge_cloud_store_backend(GXCloudRESTResource.CHECKPOINT)
-
-    with mock.patch("requests.Session.get", autospec=True) as mock_get:
-        store_backend.list_keys()
-        mock_get.assert_called_with(
-            mock.ANY,  # requests.Session object
-            f"{CLOUD_DEFAULT_BASE_URL}organizations/51379b8b-86d3-4fe7-84e9-e1a52f4a414c/checkpoints",
-        )
-
-
-@pytest.mark.cloud
-@pytest.mark.unit
-def test_remove_key(
-    construct_ge_cloud_store_backend: Callable[
-        [GXCloudRESTResource], GeCloudStoreBackend
-    ],
+def test_ge_cloud_store_backend_init_raises_deprecation_warning(
```

Comment:
This is the one unit test I've added here - just thought I'd call it out as the diff is a bit hard to understand